cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,179 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedCinemaAPITests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_unauthenticated_user_cannot_access_movie_list(self):
+        res = self.client.get(MOVIE_URL)
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
Code Quality Issue: This test case is correctly checking that an unauthorized request is rejected. However, it could be improved by also verifying that an appropriate error message is returned in the response.